docs/04.guides/04.cookbooks/33.Hidden_gems/page.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Hidden Gems
+id: hidden_gems
+---
+## Hidden Gems ##
+
+This document explains how to declare a variables, function call with dot and bracket notation, Passing arguments via URL/form scopes as array. Explain this three with simple example below:
+
+#### Example 1 : Declare variables ####
+
+// test.cfc
+
+```luceescript
+component {
+	function getName() {
+		return "Susi";
+	}
+}
+```
+
+// example1.cfm
+
+```luceescript
+function test() {
+	var qry;
+	dump(qry);
+	query name="qry" datasource="test" {
+		echo("select 1 as one");
+	}
+	dump(qry);
+}
+test();
+```
+
+* In cfm page have test() function with local variable scope, it normally assigned in empty string ``var qry``. Then execute this cfm, the qry is returns "1". 
+* Then dump the ``qry`` below of the var declaration. It returns empty string. 
```

Comment:
***ToChange***
***Lines 35 - 36***
* In the cfm page, we have the function, test(), with a local variable scope. It normally assigns the variable ``var qry`` to an empty string. When executed, this cfm returns "1" for the variable qry. 
* When we dump the ``qry`` below the variable declaration, it returns an empty string.